recipes/nyxus/bld.bat

```diff
@@ -0,0 +1,2 @@
+set CMAKE_ARGS="-DCMAKE_PREFIX_PATH=$PREFIX -DCMAKE_INSTALL_PREFIX=$PREFIX -DUSEGPU=ON" 
```

Comment:
if it's possible to build without GPU support, it can often be worth it... it can be a bit tricky, though, to handle the metapackages properly...

Reply:
So, we want to build with CUDA support eventually. What is the best way to do that. I am thinking about adding cudatoolkit-dev as a build dependency and cudatoolkit as run dependency.

Reply:
Do not use cudatoolkit-dev in a recipe. Just add `compiler('cuda')` to the build requirements section. Add the cudatoolkit to your host requirements if you need the CUDA libraries.